packages/decoupled/src/logger/log-format.ts

```diff
@@ -1,25 +1,47 @@
 import { format } from 'winston';
+import { get } from 'lodash';
+import moment from 'moment';
 import { colorizeLabel } from './colorize-label';
 
 let maxLabelLength = 0;
 
 const updateMaxLabelLength = (label: string) => maxLabelLength = Math.max(maxLabelLength, label.length);
 
 const logFormatter = format.printf((info) => {
+    let label = info.label || '';
+    const timestamp = info.timestamp || '';
+    const level = info.level ? `${info.level} : ` : '';
+    return `${label}${timestamp}${level}${info.message}`;
+});
 
-    updateMaxLabelLength(info.label);
+const setFormat = format((info, opts) => {
+    // check option show label
+    if(get(opts, 'options.showLabel', true)) {
+        updateMaxLabelLength(opts.siteId);
+        info.label = `${colorizeLabel(opts.siteId, opts.siteId.padEnd(maxLabelLength))} | `;
+    }
 
-    const label = colorizeLabel(info.label, info.label.padEnd(maxLabelLength));
-    const date = new Date(info.timestamp).toLocaleDateString('en-GB');
-    const time = new Date(info.timestamp).toLocaleTimeString('en-GB', { hour12: false });
+    // check option timestamp, set format datetime if there is  a string
+    if(get(opts, 'options.timestamp', true)) {
+        let formatDateTime = 'd/MM/YYYY hh:mm:ss';
```

Comment:
@friendken The day-format should be `DD` instead of `d`, see https://momentjs.com/docs/#/displaying/
I also suggest putting this string in a `const DEFAULT_TIME_FORMAT = "..."` in the header of the file to avoid duplicated allocation.